Userland/Applications/Calendar/main.cpp

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "AddEventDialog.h"
+#include "LibCore/System.h"
```

Comment:
Headers from external libraries are usually included using the `<>` syntax.

Reply:
I blame CLion's auto-include :D